cis_enforcements/azure/cis_v200/section9/policies.tf

```diff
@@ -0,0 +1,172 @@
+# Azure > AppService > WebApp > Approved
+resource "turbot_policy_setting" "azure_appservice_webapp_approved" {
+  resource = turbot_smart_folder.main.id
+  type     = "tmod:@turbot/azure-appservice#/policy/types/webAppApproved"
+  note     = "Azure CIS v2.0.0 - Control: 9.1, 9.6, 9,7 and 9.8"
+  value    = "Check: Approved"
+  # value    = "Enforce: Delete unapproved"
+}
+
+# Azure > AppService > WebApp > Approved > Custom
+resource "turbot_policy_setting" "azure_appservice_webapp_approved_custom" {
+  resource       = turbot_smart_folder.main.id
+  type           = "tmod:@turbot/azure-appservice#/policy/types/webAppApprovedCustom"
+  note           = "Azure CIS v2.0.0 - Control: 9.1, 9.6, 9,7 and 9.8"
+  template_input = <<-EOT
+    {
+      outdatedJavaVersions: constant(value: "['8', '11']")
+      outdatedPhpVersions: constant(value: "['5.6', '7.0', '7.1', '7.2', '7.3']")
+      outdatedPythonVersions: constant(value: "['2.7', '3.4', '3.5', '3.6']")
+      webApp {
+        appServiceAuth: get(path:"authSettings.enabled"),
+        javaVersion: get(path:"configuration.javaVersion"),
+        phpVersion: get(path:"configuration.phpVersion"),
+        pythonVersion: get(path:"configuration.pythonVersion")
+      }
+    }
+  EOT
+  template       = <<-EOT
+    {% set results = [] -%}
+
+    {%- if $.webApp.appServiceAuth == false -%}
+
+      {%- set data = {
+          "title": "App Service authentication",
+          "result": "Not approved",
+          "message": "App Service authentication is disabled"
+      } -%}
+
+    {%- else -%}
+
+      {%- set data = {
+          "title": "App Service authentication",
+          "result": "Approved",
+          "message": "App Service authentication is enabled"
+        } -%}
```

Comment:
```suggestion
    {%- elif $.webApp.appServiceAuth == true -%}

      {%- set data = {
          "title": "App Service Authentication",
          "result": "Approved",
          "message": "App Service authentication is enabled"
        } -%}
        
    {%- else -%}

      {%- set data = {
          "title": "App Service Authentication",
          "result": "Skip",
          "message": "No data for authentication yet"
        } -%}        
```

Reply:
The value of `authSettings.enabled` will be either true or false. So `no data` condition will not occur.